web/src/compose_recipient.js

```diff
@@ -3,17 +3,22 @@
 import $ from "jquery";
 import _ from "lodash";
 
+// todo: fix circular import here
```

Comment:
@timabbott what would you recommend? My best guess is to move `show_compose_box` out of `compose_actions` but that doesn't feel right, but I also can't think of anything else that seems reasonable.

Reply:
I think that's the right high-level concept, but we should do it by extracting a helper. The line `$("#compose").css({visibility: "visible"});` belongs in `compose_actions.show_compose_box`, but the rest of that function and its focus area related helpers should move to this module, with a new function name like `compose_recipient.update_compose_for_message_type("private", opts)`

Later we'll want to refactor how `opts` works to be more explainable but that can be a follow-up.